src/state.ts

```diff
@@ -14,32 +15,31 @@ export type Events =
       data: DataResponse;
     }
   | {
-      type: "search set";
+      type: "search updated";
```

Comment:
i get why you've done this, would you say "set" if you only made the update once? whereas obviously search can change lots of times during the apps use